frontend/src/employee-frontend/components/unit/unit-reservations/ChildDayAttendance.tsx

```diff
@@ -94,14 +90,19 @@ export default React.memo(function ChildDayAttendance({
       dateInfo.time === null ||
       dateInfo.isHoliday ||
       (reservation.type === 'TIMES' &&
-        dateInfo.time.start > reservation.startTime)
+        !dateInfo.time.includesStartOf(
```

Comment:
Check if `includesStartOf`/`includesEndOf` could be just `includes(foo: MidnightAwareTime)`